src/main/kotlin/com/dcd/server/core/domain/domain/service/impl/GenerateHttpConfigServiceImpl.kt

```
@@ -1,31 +1,34 @@
         val configFileName = "${application.name.replace(" ", "-")}-http.conf"
-        val commandResult = commandPort.executeShellCommand(
-            "mkdir -p '$httpConfigDirectory' && " +
-            "cat <<'EOF' > '$httpConfigDirectory/$configFileName'\n${webServerConfig}\nEOF"
-        )
+        val configFilePath = configDirectory.resolve(configFileName)
 
-        if (commandResult.exitValue != 0)
+        try {
+            fileOperationPort.createDirectory(configDirectory)
+            fileOperationPort.writeFile(configFilePath, webServerConfig)
+        } catch (e: FileOperationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)